### PR TITLE
@alloy => Always use smallest screen dimension for iphone width 

### LINKED ISSUE #505: [iPhone] ArtworkVC rotation removes the insets for the related information under the artwork
Easily reproducible by rotating into a VIR on an artwork, e.g. https://www.artsy.net/artwork/jacqueline-fowler-amari-vase-and-white-japonica

### Before

screen_shot_2015-06-04_at_9 11 34_am

### After

screen_shot_2015-06-04_at_9 11 39_am

### PR DESCRIPTION
**Pull request by orta**
Fixes #505

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -118,13 +118,13 @@ beta: stamp_date deploy
 
 
 LOCAL_BRANCH = $(shell git rev-parse --abbrev-ref HEAD)
-BRANCH = $(shell echo host=github.com | git credential fill | sed -E 'N; s/.*username=(.+)\n?.*/\1/')-$(shell git rev-parse --abbrev-ref HEAD)
+BRANCH = $(shell echo $(shell whoami)-$(shell git rev-parse --abbrev-ref HEAD))
 
 pr: 
-	if [ "$(BRANCH)" == "master" ]; then echo "In master, not PRing"; else git push upstream "$(LOCAL_BRANCH):$(BRANCH)"; open -a "Google Chrome" "https://github.com/artsy/eigen/pull/new/artsy:master...$(BRANCH)"; fi
+	if [ "$(LOCAL_BRANCH)" == "master" ]; then echo "In master, not PRing"; else git push upstream "$(LOCAL_BRANCH):$(BRANCH)"; open -a "Google Chrome" "https://github.com/artsy/eigen/pull/new/artsy:master...$(BRANCH)"; fi
```
Comment (ashfurrow) [inline Makefile]: Why specify Chrome?

File: Artsy/Classes/View Controllers/ARArtworkMasonryModule.m [hunk 2]
```diff
@@ -76,15 +76,15 @@ + (NSInteger)rankForlayout:(ARArtworkMasonryLayout)layout
 + (CGFloat)dimensionForlayout:(ARArtworkMasonryLayout)layout useLandscapeValues:(BOOL)useLandscapeValues
 {
     CGRect screenRect = [[UIScreen mainScreen] bounds];
-    CGFloat width = CGRectGetWidth(screenRect);
+    CGFloat shortestDimension = MIN(CGRectGetWidth(screenRect), CGRectGetHeight(screenRect));
```
Comment (ashfurrow) [inline Artsy/Classes/View Controllers/ARArtworkMasonryModule.m]: If you were a luxury brand, your tagline would be "Hacky. Elegant. _Orta_."

File: Makefile (hunk 1 above)
Comment (alloy) [inline Makefile]: @orta uses Chrome because he can use a plugin to make a GIF for his PR :smile:

Comment (ashfurrow) [inline Makefile]: Pfft – he can make that his default browser :P I don't even have it installed.

**Comment by alloy** [comment]
@orta Can we have a CHANGELOG to make this already excellent PR top-notch?

File: Makefile (hunk 1 above)
Comment (orta) [inline Makefile]: chrome as default, jeez Ash, what kind of person do you think I am?

**Comment by orta** [comment]
Changelog'd

**Comment by alloy** [comment]
:+1:

File: Artsy/Classes/View Controllers/ARArtworkMasonryModule.m (hunk 2 above)
Comment (orta) [inline Artsy/Classes/View Controllers/ARArtworkMasonryModule.m]: :dancers:

**Pull request merged**